### ISSUE TITLE
Wrong curl generation for working POST request

### ISSUE THREAD

**Issue by Ducane** [a85/POSTMan-Chrome-Extension#10581]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the Issue

Hello everyone,

If you create a POST Request with `Content-Type: application/x-www-form-urlencoded` filled with some form parameters and the response returns a 30x redirect response, then the executed postman request will work. The generated cURL command however is not working when pasted into a terminal.

E.g. 
```
curl --location --request POST '<URL>' \
         --header 'Content-Type: application/x-www-form-urlencoded' \
         --data-urlencode 'username=<USERNAME>' \
         --data-urlencode 'password=<PASSWORD>'
```

is being generated. However this cURL request doesn't work in the terminal since according to [this](https://github.com/curl/curl/issues/578) there is no need to specify `--request POST` if `-d`, `--data` or `--data-urlencode` is used. If you still specify `--request POST` all redirections triggered by `--location` are sent as POST methods (instead of the usual expected GET) which leads to unexpected behaviors.

The correct cURL looks like this:

```
curl --location '<URL>' \
         --header 'Content-Type: application/x-www-form-urlencoded' \
         --data-urlencode 'username=<USERNAME>' \
         --data-urlencode 'password=<PASSWORD>'
```

Since a user expects  the cURL command to  behave the same way as postman is responding i suggest removing the `--request POST` in the generated cURL.

### Steps To Reproduce

1. Create a POST at a resource `<URL>` which returns a 30x as response.
2. Add the Request-Header `Content-type: application/x-www-form-urlencoded` or check the `x-www-form-urlencoded` radio button for the body and add some key/value pairs.
4. Copy the generated curl and try to execute it in the terminal. It will try to call the redirects with POST as a method, which can lead to unexpected behavior.

### Environment Information

```markdown
- Operating System: Mac OS
- Platform Type: 
- Postman Version: v.9.0.9
```

**Comment by dhwaneetbhatt** [comment]
@Ducane We've fixed this issue in the latest version (10.10.x). Could you update to the latest version and check if it solves your problem?

**Comment by dhwaneetbhatt** [comment]
Closing this issue as it has been fixed in version 10.10. Feel free to reopen if the issue persists.
